extensions/project-descriptor.md

```diff
@@ -8,80 +8,120 @@ A project descriptor is a file that MAY contain configuration for apps, services
 - [Project Descriptor](#project-descriptor)
   - [Table of Contents](#table-of-contents)
   - [Schema](#schema)
-  - [`[project]`](#project)
-  - [`[[project.licenses]]`](#projectlicenses)
-  - [`[build.include]` and `[build.exclude]`](#buildinclude-and-buildexclude)
-  - [`[[build.buildpacks]]`](#buildbuildpacks)
-  - [`[[build.env]]`](#buildenv)
-  - [`[metadata]`](#metadata)
+    - [`[_]`](#_)
+      - [`[_]`](#_-1)
+      - [`[[_.licenses]]`](#_licenses)
+      - [`[_.metadata]`](#_metadata)
+    - [Non-`[_]` Tables](#non-_-tables)
+    - [`[io.buildpacks]`](#iobuildpacks)
+      - [`[io.buildpacks.build.include]` and `[io.buildpacks.build.exclude]`](#iobuildpacksbuildinclude-and-iobuildpacksbuildexclude)
+      - [`[[io.buildpacks.build.buildpacks]]`](#iobuildpacksbuildbuildpacks)
+      - [`[[io.buildpacks.build.env]]`](#iobuildpacksbuildenv)
```

Comment:
I think we should take the square brackets out of the headings and just include the key names. Also, do we need to repeat the parent table names if we are nesting these appropriately? E.g. could it look like:
- `io.buildpacks`
  - `api`
  - `build`
    - `buildpacks`
    - `env`
    - `exclude`
    - `include`